src/SFA.DAS.ApplyService.Web/Infrastructure/Interfaces/IProcessPageFlowService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApplyService.Web.Infrastructure.Interfaces
+{
+    public interface IProcessPageFlowService
+    {
+        Task<string> GetIntroductionPageIdForSection(Guid applicationId, int sequenceId, int providerTypeId);
```

Comment:
This accepts a sequenceId, would it be more appropriate to name is GetIntroductionPageIdForSequence?

Reply:
yep!